_posts/0000-01-02-MuskaanMLSA.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
I will work very hard to get promoted from Beta MLSA to Gold MLSA. Thank you for giving me this opportunity.